Cargo.toml

```diff
@@ -28,7 +29,7 @@ tonic012 = { package = "tonic", version = "0.12", optional = true }
 [dev-dependencies]
 sha2 = "0.10.6"
 rand = "0.8.5"
-tokio = { version = "1", features = ["macros", "rt", "io-util"] }
+tokio = { version = "1.34", features = ["macros", "rt", "io-util"] }
```

Comment:
Why don't use 1.27 wrote in the description? In my opinion, use a lower version will have less impact on user code.

Reply:
Tonic 0.12 requires tokio 1.34, so 1.27 is incompatible with the tonic012 feature